pytest_kubernetes/options.py

```diff
@@ -7,4 +7,5 @@ class ClusterOptions:
     api_version: str = field(default="1.25.3")
     # nodes: int = None
     kubeconfig_path: Path | None = None
+    cluster_config: Path | None = None  # Path to a Provider cluster config file
```

Comment:
Idea: may we rename that symbol to something more descriptive, for example `provider_cluster_config`?
(I would prefer something more concise, tho)

Reply:
I would go with `provider_config` then. As it is a provider specific config file.